client/src/container/Leads.jsx

```diff
@@ -117,7 +78,9 @@ class Leads extends React.Component {
 }
 
 const mapStateToProps = state => ({
-  leads: state.leadsReducer.leads
+  leads: state.leadsReducer.leads,
+  leadsColums: state.leadsReducer.leadsColums,
+  leadsColumsHeader: state.leadsReducer.leadsColumsHeader
```

Comment:
Did you intentionally name the property leadsColumsHeader (without an 'n' at end of column)?   All other properties and variable names you use (outside of reducer and mapStateToProps) that contain the word 'column' spell it correctly, with an N at the end.